staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -758,7 +758,7 @@ type ServerAddressByClientCIDR struct {
 	ServerAddress string `json:"serverAddress" protobuf:"bytes,2,opt,name=serverAddress"`
 }
 
-// GroupVersion contains the "group/version" and "version" string of a version.
+// GroupVersionForDiscovery contains the "group/version" and "version" string of a version.
```

Comment:
Please update the auto-generated files as well.

Reply:
@dixudx Thanks for your feedback. Could you please tell me how to update the auto-generated files? I had run `hack/update-generated-docs.sh`.

Reply:
@huangjiuyuan run `make generated_files`

Reply:
Related proto and json files has to be updated. Please run `./hack/update-all.sh` instead.

Reply:
@dixudx I see. A `generated.proto` file had been updated. Good to go?